### PR TITLE
Added Footer

### PR DESCRIPTION
**Pull request by iagocarvalhodev**
creating the footer component, using the svg icons

### CODE REVIEW

**Comment by fabriciomirandabr** [comment]
Change Title: Added Footer

Color Wrong
Space Wrong
Font Wrong
Size Wrong

Should be PERFECT PIXEL Like Figma

image

image

**Comment by fabriciomirandabr** [comment]
Code Pattern OK

File: src/components/shared/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 14px;
+    line-height: 22px;
+    color: #000000;
```
Comment (fabriciomirandabr) [inline src/components/shared/footer.tsx:86]: Colors should come from styles/variables.ts

**Review by fabriciomirandabr — CHANGES_REQUESTED** [review]
The layout should be perfect pixel

File: src/components/shared/footer.tsx (hunk 1 above)
Comment (iagocarvalhodev) [reply src/components/shared/footer.tsx]: done

**Comment by fabriciomirandabr** [comment]
image
Font import in storybook it's no right

**Comment by fabriciomirandabr** [comment]
The size of   pipe it's not right

image

**Comment by fabriciomirandabr** [comment]
The box size in large screen it's not right

image

**Comment by fabriciomirandabr** [comment]
Missformated in the tablet mode

image

**Comment by fabriciomirandabr** [comment]
Missformated in large mobile

image

**Comment by fabriciomirandabr** [comment]
Missformated in small mobile

image

File: public/index.html [hunk 2]
```diff
@@ -20,6 +20,8 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/icons/favicon-16x16.png" />
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap" rel="stylesheet" />
     <link rel="manifest" href="/icons/manifest.json" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
```
Comment (fabriciomirandabr) [inline public/index.html:23]: What is this?

**Pull request closed without merging**